### DISCUSSION TITLE
Transitions in Segue.

### DISCUSSION THREAD

**Discussion by jem5519** [SwiftfulThinking/SwiftfulRouting#30] [Q&A] [answered]
Does the framework support tranitioons in Segues? 

if so, how?

**Comment by SwiftfulThinking** [comment, marked as answer]
Is supports Modals which use Transitions. Is that what you mean? Native SwiftUI segues are not technically Transitions.Sent from my iPhoneOn Sep 26, 2023, at 9:05 AM, Jim Malak ***@***.***> wrote:﻿
Does the framework support tranitioons in Segues?
if so, how?

**Reply by jem5519** [reply]
I see the examples on screen 0 the sample project. Many thanks!
